src/Controls/tests/BindingSourceGen.UnitTests/ObservablePropertyTests.cs

```diff
@@ -99,9 +102,12 @@ public void Test()
 		// The binding should be generated successfully with ObservableProperty inference
 		Assert.NotNull(result.Binding);
 
-		// Verify the generated interceptor code contains the Title property access
+		// Verify the generated interceptor code contains the correct getter and setter references
 		var allGeneratedCode = string.Join("\n\n", result.GeneratedFiles.Values);
-		Assert.Contains("Title", allGeneratedCode, System.StringComparison.Ordinal);
+		// Check that the method signature has the getter parameter with correct type
+		Assert.Contains("global::System.Func<global::TestApp.MyViewModel, string?> getter", allGeneratedCode, System.StringComparison.Ordinal);
```

Comment:
This is not a useful assert. The useful variant of this would be `(source.Title, true)`.